plugin-api.d.ts

```diff
@@ -1427,8 +1433,9 @@ interface SceneNodeMixin {
     readonly componentProperties?: {
       readonly [propertyName: string]: VariableAlias
     }
+    readonly textRangeFills?: VariableAlias[]
```

Comment:
Just to clarify, this should be here right?

Reply:
uhhh i'm just running `npm run update-typings`

Reply:
[Original PR](https://github.com/figma/figma/pull/206278)